package.json

```diff
@@ -55,11 +55,6 @@
       "docs/"
     ]
   },
-  "babel": {
```

Comment:
I just deleted it locally and it seems like it still rolls up fine, so I guess delete it?